spec/models/trainee_withdrawal_reason_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe TraineeWithdrawalReason do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
I think it's ok to delete this file rather than have a `pending` spec. The model is logicless so nothing to test. Same for next one below...

Reply:
this is populated now with some basic specs